addons/bestja_detailed_reports/models.py

```diff
@@ -171,6 +198,28 @@ def _compute_project_dates(self):
         return "{} — {}".format(date_start, date_stop)
 
     @api.one
+    @api.depends('project', 'report_entries')
+    def _compute_total_cities_nr(self):
+        """
+            Computes number of unique cities where stores
+            are held, both for the organization and all its children.
```

Comment:
Maybe a quick comment explaining why partner organisations always have "0" in this field?